### PR TITLE
Updates to the Board page for the 2023 Board

### PR DESCRIPTION
**Pull request by spier**

### CODE REVIEW

**Comment by spier** [comment]
@mcobby for this to go live I am only missing your profile picture and a brief bio of yourself. See [this](https://innersourcecommons.org/about/board/) for reference of what the others have written.

You can send this to me via Slack, and I can integrate it into this PR. Or you can contribute to this PR directly, whatever is faster for you.

cc @tsadler1988

File: content/en/about/board/_index.md [hunk 1]
```diff
@@ -18,42 +18,46 @@ aliases:
...
 {{< /board-member >}}
 
-{{< board-member name="Jacob Green" image="/images/about/Jacob_Green.jpg" style="bg-light" >}}
-Jacob Green, the founder of Mosslabs.io, is an open source artist, strategist & systems builder working to expand the impact and application of Open Source in society globally. Hailing from Baltimore, passionate about open source in cities & universities, and working to connect institutions via open source. Prior to Mosslabs.io, Jacob was Director of Strategic Initiatives at LTNGlobal, building a unique global video transport network from the ground up and from first principles.
+{{< board-member name="Georg Grütter" image="/images/about/Georg_Grutter.jpeg" style="bg-light" >}}
+Georg Grütter is a social coding evangelist and developer advocate at Bosch.IO. He co-founded and led the first InnerSource community at Bosch. Georg is a passionate software developer with over 30 years of experience. Previously, he held various positions and roles at Bosch, Line Information, the Zurich System House, and DaimlerChrysler. Georg has created two open source projects, who also loves photography and chocolate.
```
Comment (lenucksi) [inline content/en/about/board/_index.md]: That sentence was grammatically broken.

```suggestion
Georg Grütter is a social coding evangelist and developer advocate at Bosch.IO. He co-founded and led the first InnerSource community at Bosch. Georg is a passionate software developer with over 30 years of experience. Previously, he held various positions and roles at Bosch, Line Information, the Zurich System House, and DaimlerChrysler. Georg has created two open source projects, and also loves photography and chocolate.
```

Comment (spier) [reply content/en/about/board/_index.md]: Thanks for the fix!

File: content/en/about/board/_index.md [hunk 2]
```diff
@@ -66,26 +73,34 @@ Tom Sadler is a Software Engineering Team Lead for BBC iPlayer & Sounds, working
...
 {{< /board-member >}}
 
-{{< board-member name="Tim Yao" image="/images/about/Tim_Yao.jpeg" >}}
-Tim is a Distinguished Member of Technical Staff and the Product Manager for Nokia Continuous Delivery in Nokia Cloud and Network Services. Over his twenty-plus year career in telecommunications, Tim has held roles in innovation, software testing, system engineering, architecture, and procurement. He served six years on the Alcatel-Lucent FOSS Executive Committee. Tim has experience creating grass roots communities within the company and outside of it; he is a volunteer co-Municipal Liaison for National Novel Writing Month.
+{{< board-member name="Danese Cooper" title="Chairperson" image="/images/about/Danese_Cooper.jpg" >}}
```
Comment (rrrutledge) [inline content/en/about/board/_index.md]:
```suggestion
{{< board-member name="Danese Cooper" image="/images/about/Danese_Cooper.jpg" >}}
```

**Review by rrrutledge — CHANGES_REQUESTED** [review]
Only include current director roles.

**Comment by spier** [comment]
> Only include current director roles.

Great catch @rrrutledge, thank you!

**Review by rrrutledge — APPROVED** [review]
Thanks for updating, @spier ❗

**Pull request merged**